### PR TITLE
feat: added limit-req plugin form

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Add limit-req plugin form.
image

**Related issues**

None

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <Form
+      form={form}
+      {...FORM_ITEM_LAYOUT}
+    >
+      <Form.Item
+        label="rate"
+        name="rate"
+        required
+      >
+        <InputNumber min={1} required></InputNumber>
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/limit-req.tsx:46]:
```suggestion
        <InputNumber min={1} required />
```

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        <InputNumber min={0} required></InputNumber>
+      </Form.Item>
+      <Form.Item
+        label="key"
+        required
+        name="key"
+      >
+        <Select>
+          {["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"].map(item => {
+            return <Select.Option value={item}>{item}</Select.Option>
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/limit-req.tsx:62]:
```suggestion
            return <Select.Option value={item} key={item}>{item}</Select.Option>
```

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        required
+      >
+        <InputNumber min={1} required></InputNumber>
+      </Form.Item>
+      <Form.Item
+        label="burst"
+        name="burst"
+        required
+      >
+        <InputNumber min={0} required></InputNumber>
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/limit-req.tsx:53]:
```suggestion
        <InputNumber min={0} required />
```

File: web/src/components/Plugin/UI/limit-req.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            return <Select.Option value={item}>{item}</Select.Option>
+          })}
+        </Select>
+      </Form.Item>
+      <Form.Item
+        label="rejected_code"
+        name="rejected_code"
+        initialValue={503}
+      >
+        <InputNumber min={200} max={599}></InputNumber>
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/limit-req.tsx:71]:
```suggestion
        <InputNumber min={200} max={599} />
```

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Please file a new PR to add Fields Descriptions in tooltip

**Review by LiteSun — APPROVED** [review]

**Review by Yiyiyimu — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@guoqqqi conflicts

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Please update i18n in a new PR @guoqqqi

**Comment by guoqqqi** [comment]
OK😆

**Review by starsz — APPROVED** [review]

**Pull request merged**